### DISCUSSION TITLE
Provide possibility to download to pdf a section

### DISCUSSION THREAD

**Discussion by Sotty75** [facebook/docusaurus#9235] [Ideas]
In my company i am often asked to provide a PDF of a given part of the documentation site.
Similarly to what is done in other sites, it would be nice to provide a download to PDF function naitvely which is avalable on a page. 
The PDF generated may contain the current page and any page which is under the current page in the sidebar hierarchy.

**Comment by slorber** [comment]
There are many issues discussion pdf + Docusaurus already, like this one:
https://github.com/facebook/docusaurus/issues/969

We don't provide that functionality and it would be complicated to implement considering Docusaurus builds a static site and does not have a server runtime: we would need to pre-build all the possible pdf variations at build time.

The best option you have is to create your own dynamic pdf generation runtime and then add a "download to pdf button" using Docusaurus swizzle
